src/engine.pure.cjs

```diff
@@ -15,6 +15,12 @@ let willstart
 
 const abstractProcess = globalThis.process || globalThis.EXODUS_TEST_PROCESS
 
+if (process.env.EXODUS_TEST_IS_BROWSER) {
+  let promise
+  globalThis.EXODUS_TEST_PROMISE = new Promise((resolve, reject) => (promise = { resolve, reject }))
+  abstractProcess._exitHook = (exitCode) => promise.resolve(exitCode)
```

Comment:
can we not assign `abstractProcess._exitHook` inside the promise callback (so we don't have to assign to `promise`)?

Reply:
Yes, thanks!

That was a chain of refactors and I failed to notice that in the final form `promise` wasn't needed at all

Done: 7ad8af91fa216dbc3d3068cf599a278d40e8d294